### DISCUSSION TITLE
I can't find the TumblThree.exe file?

### DISCUSSION THREAD

**Discussion by Sara-Abo-Shousha** [TumblThreeApp/TumblThree#335] [Q&A (Question & Answer)]
like why? i downloaded it from here

**Comment by thomas694** [comment]
The file ("TumblThree-v1.5.2-x64-Application.zip") you downloaded is a zip archive, you have to extract it first. Inside the extracted folder there is the app TumblThree.exe.

**Comment by thomas694** [comment]
The issue has been closed. You can still comment. Feel free to ask for reopening the issue if needed.

**Reply by neubauje** [reply]
I'm requesting this be reopened, because I have the same question. I extracted the zip file, which contained a single folder (TumblThree-master). Within that folder, I see the following:

.github
docs
lib
scripts
src
.gitattributes
.gitignore
appveyor.yml
Contributing.md
LICENSE
LICENSE-3RD-PARTY
README.md

I do *not* see any .exe files in the contents of this zip file.

**Reply by thomas694** [reply]
This was the zip file for developers under the Code button.

As metioned on the main page ("Latest versions can be found [here](https://github.com/TumblThreeApp/TumblThree/releases).") versions can be found on the right side under "[Releases](https://github.com/TumblThreeApp/TumblThree/releases)" (e.g. TumblThree-v2.11.4-x64-Application.zip).
